src/AppInstallerCommonCore/Manifest/ManifestValidation.cpp

```diff
@@ -331,6 +334,15 @@ namespace AppInstaller::Manifest
                         resultErrors.emplace_back(ManifestError::DuplicatePortableCommandAlias, "PortableCommandAlias");
                         break;
                     }
+
+                    // If running full validation, check filetype
+                    if (fullValidation && isPortable)
+                    {
+                        if (fullPath.has_extension() && allowedPortableFiletypes.find(fullPath.extension()) == allowedPortableFiletypes.end())

```

Comment:
Do we want to allow .ExE?

Reply:
Sorry, never mind. After reading the implementation code, it only supports .exe.  But, the code gets the final name from either PortableCommandAlias or RelativeFilePath. So the alias should be checked against ".exe" as well (if it has extention).

Reply:
Since Windows is case-insensitive, I think it makes sense to force `.exe` here, though I could add a utility function for making paths lowercase for comparison. My main concern is avoiding the proliferation of having to have `.exe`, `.Exe`, `.eXe`, `.exE`, `.EXe`, `.ExE`, `.eXE`, and `.EXE` all in the allow list

Reply:
Yes, I think we can leave the code as is.